source/simulator/solver_schemes.cc

```diff
@@ -1299,9 +1143,35 @@ namespace aspect
                       (1.0-(dcr.newton_residual_for_derivative_scaling_factor/dcr.switch_initial_residual))));
 
         do_one_defect_correction_Stokes_step(dcr, use_picard);
-        relative_residual = dcr.residual/dcr.initial_residual;
 
-        pcout << std::endl;
+        double max = 0.0;
```

Comment:
Probably like this before the change, but it would be easier if you did this:
```suggestion
        double max_residual = relative_temperature_residual;
```
and then delete line 1170. This brings the computation of the maximum residual in the same order as the output of the residuals.